app/views/ControlWindow.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @author Bingzheng Feng
+ * @module views/ControlWindow
+ * @requires models/Commands
+ * @requires util/Validator
+ * @requires util/Logger
+ * @requires models/TelemetryData
+ * @requires util/Template
+ * @requires electron
+ * @listens models/TelemetryData~TelemetryData:data_received
+ * @copyright Waterloo Aerial Robotics Group 2016
+ * @licence https://raw.githubusercontent.com/UWARG/WARG-Ground-Station/master/LICENSE
+ * @description Responsible for displaying control pitch, yaw, roll
+ */
+
+var remote = require('electron').remote;
+var Template = require('../util/Template');
+var TelemetryData = remote.require('./app/models/TelemetryData');
+var Logger = remote.require('./app/util/Logger');
+var Validator = require('../util/Validator');
+var Commands = remote.require('./app/models/Commands');
+var Channel_Conf = remote.require('./config/vehicle-config');
+var PicPilot_Conf = remote.require('./config/picpilot-config');
+
+module.exports = function (Marionette) {
+
+  return Marionette.ItemView.extend({
+    template: Template('ControlWindow'),
+    className: 'controlWinodw',
+
+    ui: {
+      front23: '.controlWindow.circle.front.twothree',
```

Comment:
same thing as above with the naming. This is not a window, this is a view. In your case you put this view into its own window, but that isnt necessarily a restriction or necessary (we can put multiple views inside the same window)